{"name": "数据库中查不到用户", "status": "failed", "statusDetails": {"message": "AssertionError: assert '-3' == '-1'\n  - -1\n  + -3", "trace": "pyfixture = ({'appkey': 1000, 'clientIP': '192.168.10.23', 'custid': 104401903883, 'isFrontendCall': 'no', ...}, ('-1', '数据库中查不到用户'))\n\n    @allure.title('{pyfixture[1][1]}')\n    @pytest.mark.change_mobilephone\n    @pytest.mark.flaky(reruns=1,reruns_delay=10)\n    def test_ChangeMobilePhone(pyfixture):\n    \n        #请求\n        url=Change_MobilePhone.url\n        data=pyfixture[0]\n        res=request(url=url,data=data)\n        #print(data)\n    \n        #返回码检查\n>       assert res['statusCode']==pyfixture[1][0]\nE       AssertionError: assert '-3' == '-1'\nE         - -1\nE         + -3\n\ntestcase\\Loginapi\\Change_MobilePhone_test.py:126: AssertionError"}, "parameters": [{"name": "pyfixture", "value": "({'custid': 104401903883, 'mobile_phone': 18811348250, 'type': 7, 'clientIP': '192.168.10.23', 'isFrontendCall': 'no', 'operatorName': 'auto_tester', 'appkey': 1000, 'unionId': '', 'openId': ''}, ('-1', '数据库中查不到用户'))"}], "start": 1618906182621, "stop": 1618906182641, "uuid": "ba1b3d9b-f38a-4a03-9c63-52032955d19d", "historyId": "9386beeb949c17c57256825d409e1410", "testCaseId": "01cd1c9cbfa31c446f380425e509edb1", "fullName": "testcase.Loginapi.Change_MobilePhone_test#test_ChangeMobilePhone", "labels": [{"name": "tag", "value": "change_mobilephone"}, {"name": "tag", "value": "flaky(reruns=1, reruns_delay=10)"}, {"name": "parentSuite", "value": "testcase.Loginapi"}, {"name": "suite", "value": "Change_MobilePhone_test"}, {"name": "host", "value": "A06P-A90F12DAA"}, {"name": "thread", "value": "8780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.Loginapi.Change_MobilePhone_test"}]}